src/InputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+import React, { useRef, useEffect } from 'react'
+
+function InputWithLabel(props) {
+    const inputRef = useRef(null); 
+
+    useEffect(() => {
+        inputRef.current.focus();
+    }, []);
```

Comment:
Only for this exercise case you should remove the dependencies array in the useEffect in order to achieve the persistent input focus state after adding and removing todos.
If you leave the dependencies array like in your code , the focus will be applied only on InputWithLabel mounting.

Reply:
Done! @gsvidal